mobile/native/src/dlc/node.rs

```diff
@@ -540,7 +540,7 @@ impl Node {
         match self
             .inner
             .dlc_manager
-            .accept_channel(&channel_id)
+            .accept_channel(&channel_id, dlc::FeeConfig::AllOffer)
```

Comment:
🤔 Don't we need to separate between the two flows here? or don't we support the `EvenSplit` flow any more from the app?

Reply:
I think you will have to look into the offered channel and use the corresponding `FeeConfig`